src/lib/panic.c

```diff
@@ -52,7 +52,7 @@ void dump_panicinfo(void *addr, struct sof_ipc_panic_info *panic_info)
 
 /* panic and rewind stack */
 void panic_rewind(uint32_t p, uint32_t stack_rewind_frames,
-		  struct sof_ipc_panic_info *panic_info)
+		  struct sof_ipc_panic_info *panic_info, uintptr_t *epc1)
```

Comment:
ditto, rename to data in all generic code, but fine to use epc1 in xtensa code.

Reply:
done